Formula/go.rb

```diff
@@ -4,20 +4,14 @@ class Go < Formula
   license "BSD-3-Clause"
 
   stable do
-    if Hardware::CPU.arm?
-      url "https://golang.org/dl/go1.16rc1.src.tar.gz"
-      sha256 "6a33569f9d0d21db31614086cc2a4f0fbc683b41c1c53fb512a1341ce5763ff5"
-      version "1.15.8"
-    else
-      url "https://golang.org/dl/go1.15.8.src.tar.gz"
-      mirror "https://fossies.org/linux/misc/go1.15.8.src.tar.gz"
-      sha256 "540c0ab7781084d124991321ed1458e479982de94454a98afab6acadf38497c2"
-    end
-
+    url "https://golang.org/dl/go1.16.src.tar.gz"
+    mirror "https://fossies.org/linux/misc/go1.16.src.tar.gz"
+    sha256 "7688063d55656105898f323d90a79a39c378d86fe89ae192eb3b7fc46347c95a"
     go_version = version.major_minor
+
     resource "gotools" do
       url "https://go.googlesource.com/tools.git",
-          branch: "release-branch.go#{go_version}"
+          branch: "internal-branch.go#{go_version}-vendor"
```

Comment:
From [your linked comment](https://github.com/golang/go/issues/36882#issuecomment-712831596) it seems that we should actually be following `master`, and not some internal branch. Is my understanding here correct, @FiloSottile?

This would allow you to drop the `go_version` line, as it doesn't look to be in use elsewhere.

Reply:
That's correct, you should follow `master`.

`internal-branch.go1.X-vendor` (and `release-branch` before it) is just about what we vendor into the Go tree, it's not for user consumption. It does not get patches and is not required for compatibility with Go 1.X.